tests/rms/test_generate_petro_jobs_for_field_update.py

```diff
@@ -0,0 +1,404 @@
+                                "Mode": "STANDARD",
+                                "RangeAzimuth": 2500.0,
+                                "RangeAzimuthNormal": 1000,
+                                "RangeVertical": 25.0,
+                                "Type": "GENERAL_EXPONENTIAL",
+                                "GeneralExponentialPower": 1.8,
+                                "VariogramSillType": "CONSTANT",
+                                "VariogramSill": model_param_stdev,
+                            },
+                        ],
+                        "Transform Sequence": [
+                            {
+                                "EstimationMode": "FIXED",
+                                "WeightLog": "- none -",
+                                "Truncate": [
+                                    {
+                                        "Active": True,
+                                        "AppliedTo": "INPUT_AND_OUTPUT",
+                                        "Automated": False,
+                                        "Mode": "MIN",
+                                        "Min": 0,
+                                        "SequenceNumber": 0,
+                                    }
+                                ],
+                                "Mean": [
+                                    {
+                                        "Active": True,
+                                        "Automated": False,
+                                        "Mean": model_param_mean,
+                                        "SequenceNumber": 1,
+                                    }
+                                ],
+                            },
+                        ],
+                    }
+                )
+            corr_model = [{"CorrelationMatrix": [[], [0.5]]}]
+            facies_model_list.append(
+                {
+                    "FaciesName": facies_name,
+                    "Variable Models": var_model_list,
+                    "Correlation Model": corr_model,
+                }
+            )
+        zone_model_list.append(
+            {"ZoneName": zone_name, "Facies Models": facies_model_list}
+        )
+
+    return {
+        "Algorithm": simulation_mode,
+        "ConditionOnBlockedWells": False,
+        "InputFaciesProperty": ["Grid models", grid_name, facies_real_name],
+        "VariableNames": common_petro_list,
+        "Zone Models": zone_model_list,
+        "PrefixOutputName": "original",
+    }
+
+
+def rename_subgrids(xtgeo_grd):
+    nsubgrids = len(list(xtgeo_grd.subgrids.keys()))
+    subgrid_names = []
+    for i in range(nsubgrids):
+        name_of_subgrid = f"Zone{i+1}"
+        subgrid_names.append(name_of_subgrid)
+    xtgeo_grd.rename_subgrids(subgrid_names)
+
+
+# Here the temporary RMS project is created. It contains a grid,
+# a facies parameter and an original petrophysical job
+# @pytest.mark.skipunlessroxar
+# @pytest.fixture
+def create_project():
```

Comment:
I think we could make this into a pytest fixture in stead; then the teardown will be automated and the we can let the project live in `tmp_path` during test

Reply:
I will wait with this. Can be done later when tmp_path is introduced.